xbmc/addons/kodi-addon-dev-kit/doxygen/Skin/skin.dox

```diff
@@ -0,0 +1,27 @@
+/*!
+
+\page skin_parts Skin Development
+\brief \htmlonly
+      <h3><span  style="text-decoration: underline;"><span  style="font-style: italic;"><span
+               style="color: rgb(102, 102, 102);">Skin Add-On Development</span></span></span></h3>
+       \endhtmlonly
+
+Kodi includes a GUI library that allows you to skin/change everything you see in
+Kodi, from the images, the sizes and positions of all controls, colours, fonts,
+and text, through to altering navigation and even adding new functionality.
+
+The skin system is quite complex, and this portion of the manual is dedicated to
+providing in depth information on how it all works, along with tips to make the
+experience a little more pleasant.
+
+If you are just getting started with skinning Kodi, then it is suggested that the
+best way to learn is by modifying one of the many existing skins that are
+available. The default skin, Confluence, includes almost all the various tricks
```

Comment:
just  "The default skin "Estuary" includes almost all the various tricks" is fine.